### DISCUSSION TITLE
v0.0.3

### DISCUSSION THREAD

**Discussion by ajthinking** [ajthinking/data-story#142] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/ajthinking/data-story/releases/tag/v0.0.3'>v0.0.3</a>.</em>

**Comment by ajthinking** [comment]
### Installing on Mac
When opening first time:

Open Finder.
Locate the app.
Control+Click the app.
Select Open.
Click Open.
The app should be saved as an exception in your security settings, allowing you to open it in the future.

**Comment by stone-lyl** [comment]
### Installing on Windows

- download the [data-story-desktop-0.0.3-Setup.exe](https://github.com/ajthinking/data-story/releases/download/v0.0.3/data-story-desktop-0.0.3-Setup.exe) file, and ignore any browser warnings.
- Double-click `data-story-desktop-0.0.3-Setup.exe` in your download folder to start the installation.
- If a Windows Defender warning appears, select "More Information" then click "Run Anyway".
- Allow network access during the installation process.
- The installation will complete automatically, wait until it finishes.
- After installation, you can launch the application via the desktop shortcut or from the installation directory.
